ext/openssl/openssl.stub.php

```diff
@@ -127,18 +161,23 @@ function openssl_get_privatekey($private_key, ?string $passphrase = null): OpenS
  */
 function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
+/** @sensitive-param $password */
 function openssl_pbkdf2(string $password, string $salt, int $key_length, int $iterations, string $digest_algo = "sha1"): string|false {}
```

Comment:
shouldn't the `$salt` param be marked as sensitive?

Reply:
Thanks for the explanation! Then most probably you can ignore my below comments :)